library/src/hipblas_module.f90

```diff
@@ -12089,17 +12120,49 @@ function hipblasGemmBatchedEx(handle, transA, transB, m, n, k, alpha, a, a_type,
             integer(c_int), value :: batch_count
             integer(kind(HIPBLAS_R_16F)), value :: compute_type
             integer(kind(HIPBLAS_GEMM_DEFAULT)), value :: algo
+        end function hipblasGemmBatchedEx
+    end interface
+
+    interface
+        function hipblasGemmBatchedExWithFlags(handle, transA, transB, m, n, k, alpha, a, a_type, lda, &
+                                               b, b_type, ldb, beta, c, c_type, ldc, &
+                                               batch_count, compute_type, algo, solution_index, flags) &
+            bind(c, name='hipblasGemmBatchedExWithFlags')
+            use iso_c_binding
+            use hipblas_enums
+            implicit none
+            integer(kind(HIPBLAS_STATUS_SUCCESS)) :: hipblasGemmBatchedExWithFlags
+            type(c_ptr), value :: handle
+            integer(kind(HIPBLAS_OP_N)), value :: transA
+            integer(kind(HIPBLAS_OP_N)), value :: transB
+            integer(c_int), value :: m
+            integer(c_int), value :: n
+            integer(c_int), value :: k
+            type(c_ptr), value :: alpha
+            type(c_ptr), value :: a
+            integer(kind(HIPBLAS_R_16F)), value :: a_type
+            integer(c_int), value :: lda
+            type(c_ptr), value :: b
+            integer(kind(HIPBLAS_R_16F)), value :: b_type
+            integer(c_int), value :: ldb
+            type(c_ptr), value :: beta
+            type(c_ptr), value :: c
+            integer(kind(HIPBLAS_R_16F)), value :: c_type
+            integer(c_int), value :: ldc
+            integer(c_int), value :: batch_count
+            integer(kind(HIPBLAS_R_16F)), value :: compute_type
+            integer(kind(HIPBLAS_GEMM_DEFAULT)), value :: algo
             integer(c_int32_t), value :: solution_index
```

Comment:
Same here

Reply:
Sorry @daineAMD didn't really start on this yet, only wanted Nielen to sign off on deletion of solution index.  I'll push a partial implementation of testing to get your review before going on to batched & strided with it.  Only v2 could have the new API from my understanding so there is no inconsistency...